homeassistant/components/openevse/manifest.json

```diff
@@ -1,10 +1,12 @@
 {
   "domain": "openevse",
   "name": "OpenEVSE",
-  "codeowners": [],
+  "codeowners": ["@c00w"],
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/openevse",
+  "integration_type": "device",
   "iot_class": "local_polling",
   "loggers": ["openevsewifi"],
-  "quality_scale": "legacy",
+  "quality_scale": "bronze",
```

Comment:
Please change the quality scale level and record in separate PRs from any other changes, in existing integrations.

We want a separate PR for the feature change and a separate PR that records the feature in the quality scale record YAML file and a separate PR that updates the quality scale level in the manifest JSON file.

This will help speed up PR reviews.

Reply:
Done.